docs/system_overview/genquery.md

```diff
@@ -553,3 +551,331 @@ Find all data objects with replicas on resource `replResc` which are not marked
 ```
 SELECT COLL_NAME, DATA_NAME WHERE DATA_RESC_NAME = 'replResc' and DATA_REPL_STATUS != '1'
 ```
+
+## GenQuery2
+
+GenQuery2 is an experimental parser designed to give users more control over their queries.
```

Comment:
we'll have to remove 'experimental' one day when GenQuery2 becomes default... so... perhaps 'new' is good enough here?

```suggestion
GenQuery2 is an new parser designed to give users more control over their queries.
```

Reply:
"new" is fine.

The note section immediately following that line explains the experimental-ness of the parser, so we don't lose anything by making the suggested edit.

Reply:
right - i saw them later - i think there are two other mentions that we'd need to remove once we get to 'default GenQuery2'

Reply:
Correct.

Updated.